electroncash_plugins/fusion/plugin.py

```diff
@@ -598,6 +601,19 @@ def run(self, ):
                     for f in list(wallet._fusions_auto):
                         f.stop('Wallet has unconfirmed coins... waiting.', not_if_running = True)
                     continue
+
+                fuse_depth = Conf(wallet).fuse_depth
+                if fuse_depth > 0:
+                    sum_eligible_values = 0
+                    sum_fuz_values = 0
+                    for (eaddr, ecoins) in eligible:
```

Comment:
Nit: parens unnecessary

```suggestion
                    for eaddr, ecoins in eligible:
```